src/core/public/chrome/chrome_service.mock.ts

```diff
@@ -63,8 +65,6 @@ const createStartContractMock = () => {
       getRight$: jest.fn(),
     },
     setAppTitle: jest.fn(),
-    setBrand: jest.fn(),
-    getBrand$: jest.fn(),
```

Comment:
setBrand and getBrand were inherited but not used anywhere and did nothing. I removed them to avoid confusion.